src/main/java/uk/ac/ebi/eva/pipeline/parameters/AnnotationParameters.java

```diff
@@ -69,6 +69,9 @@
     @Value(PARAMETER + JobParametersNames.INPUT_FASTA + END)
     private String inputFasta;
 
+    @Value(PARAMETER + JobParametersNames.ANNOTATION_OVERWRITE + END)
+    private Boolean overwriteAnnotation;
```

Comment:
my main concern with the default values was where to define them, ensuring the defaults aren't scattered through the code. I'm fine putting them in the *Parameteres classes